src/models/Config.ts

```diff
@@ -103,9 +104,12 @@ export const baseConfig = {
     legendSets: {
         achieved: "ACTUAL_TARGET_ACHIEVED",
     },
-    organisationUnitGroups: {
+    organisationUnitGroupsPrefixes: {
         awardNumberPrefix: "AWARD_NUMBER_",
     },
+    organisationUnitGroups: {
+        isDartApplicable: "IS_DART_APPLICABLE",
```

Comment:
The OUG in the metadata JSON is has name=IS_DART_APPLICABLE. Typically we use a human name, ie "DART Applicable"